pkg/subgraph/src/mappings/registry-community.ts

```diff
@@ -51,6 +55,7 @@ export function handleInitialized(event: RegistryInitialized): void {
     newRC.alloAddress = rcc.allo().toHexString();
     newRC.isKickEnabled = rcc.isKickEnabled();
     newRC.communityFee = rcc.communityFee();
+    newRC.protocolFee = BigInt.fromI32(0);
```

Comment:
here i think is the same, since if they dont set a protocol fee the fee is 0 indeed, and help us to get the right data for the frontend to display the protocol fee and not trying to ask for a value that returns null

Reply:
you need take it from the contract then, because if we change the default that will be always zero. the value can be get from RegistryFactory